pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -5085,27 +5084,6 @@ broken-packages:
   - hakismet
   - hakka
   - hako
-  - hakyll
```

Comment:
Please don't unmark all those broken packages unless you are certain (i.e. you have personally verified) that they actually compile. I'm pretty sure that not all of those packages will build even if `hakyll` itself is fixed.

Reply:
Okay let's only mark hakyll as unbroken then. I have tested a couple others and yeah a lot of them still fail to build.

Reply:
Do broken packages get automatically added to this file? If so, then unmarking revdeps, having them fail and get re-added doesn't seem so bad.